### DISCUSSION TITLE
InertiaJS Set-Cookie header ignored

### DISCUSSION THREAD

**Discussion by atakansn** [adonisjs/core#4717] [Help] [answered]
I am writing an article with Quill text editor in adonis and inertia(vue3).
On the VineJS side I define a rule text: `vine.string().maxLength(20000).` 
When I enter 21000 characters from the editor;
`create.vue:198 Set-Cookie header is ignored in response from url: http://localhost:3333/panel/yazilar. The combined size of the name and value must be less than or equal to 4096 characters. `
 gives this warning and is not caught by the validator, but when I make another xhr library that enters the same, it is caught by the validator.

**Comment by RomainLanz** [comment, marked as answer]
Hey @atakansn! 👋🏻 

The issue lies elsewhere — when the validation rules are triggered, it stores the error and old value in the session, which then gets written to a cookie. However, your text is too large to fit into the cookie.

To fix this, you'll need to change your session driver. That should do the trick! 👍🏻

**Reply by atakansn** [reply]
Thank you, well, is there a structure you recommend about storage or should a structure like in laravel be made? As far as I know, adonis is currently storing with redis, I think.
